ogc-example/src/main/resources/application-spacetimekey.conf

```diff
@@ -0,0 +1,220 @@
+wms = {
```

Comment:
Actually this is a pretty cool idea to add this conf.  :100:

Reply:
Yeah. A bit of a caveat. I'm not sure its generated correctly to actually return images in QGis, but it at least is setup so that metadata is correctly generated by gt server. This is also the only temporal layer I've tested our new features against, so I expect bugs/inconsistencies/edge cases as we test against more. We'll likely want to make this example better as we iterate on temporal support.